tardis/tardis_portal/models/access_control.py

```diff
@@ -99,6 +100,7 @@ class UserAuthentication(models.Model):
     userProfile = models.ForeignKey(UserProfile, on_delete=models.CASCADE)
     username = models.CharField(max_length=50)
     authenticationMethod = models.CharField(max_length=30, choices=CHOICES)
+    approved = models.BooleanField(default=False)
 
     class Meta:
         app_label = 'tardis_portal'
```

Comment:
If we're introducing a migration for the `UserAuthentication` model, it might be worth considering combining it with https://github.com/mytardis/mytardis/issues/678 i.e. add a line to the `class Meta:` section, saying:

`unique_together = ['userProfile', 'authenticationMethod']`

OR

`unique_together = ['username', 'authenticationMethod']`